README.md

```diff
@@ -48,14 +54,18 @@ module.exports = {
             use: [
                 "style-loader", // creates style nodes from JS strings
                 "css-loader", // translates CSS into CommonJS
-                "sass-loader" // compiles Sass to CSS
+                {
+                  loader: "sass-loader", // compiles Sass to CSS
+                  // require("node-sass") instead to use Node Sass
+                  options: {implementation: require("sass")}
```

Comment:
Let's left default example as is and improve section about `dart-sass`
Like this:
```
<h2 align="center">Examples</h2>
### `node-sass` default
// Description and example
### `dart-sass`
// Description and example
```

###

Reply:
@nex3 please do this, we switch on dart sass in next major release. We should try dart sass in wild before release major.